lib/internal/bootstrap/node.js

```diff
@@ -578,6 +578,25 @@
       // It's possible that defaultTriggerAsyncId was set for a constructor
       // call that threw and was never cleared. So clear it now.
       clearDefaultTriggerAsyncId();
+      // if node-report is enabled, call into its handler to see
+      // whether it is interested in handling the situation.
+      // Ignore if the error is scopped inside a domain.
```

Comment:
```suggestion
      // Ignore if the error is scoped inside a domain.
```

Reply:
fixed, thanks.

Reply:
I’m not seeing this anywhere? I mean, it’s just a typo, but it would be nice to address it :)